edx-solutions/edx-platform
# [MCKIN-21879] Remove tables from deprecated `progress-edx-platform-extensions`

The `progress-edx-platform-extensions` has been deprecated in favor of `edx-completion`.

The requirement was removed in [this commit](https://github.com/edx-solutions/edx-platform/commit/59bf3efe71533de53b60bd979517e889d18a96bb) below. However its [migration](https://github.com/edx-solutions/progress-edx-platform-extensions/blob/master/progress/migrations/0001_initial.py) had not been reverted.
That migration used `auth_user.id` as the foreign key in its [models](https://github.com/edx-solutions/progress-edx-platform-extensions/blob/master/progress/models.py), but Django does not resolve this constraint
between existing tables anymore, because the model has been removed.
Therefore we need to drop the tables related to deprecated application in order to be able to remove users properly.

**Testing instructions**:
1. Run `./manage.py lms migrate`.
1. Go to Django admin and delete one of the users with id `6, 7, 9, 16 or 29` (if you have the same test DB that I do; otherwise you can run `SELECT user_id FROM progress_coursemodulecompletion;` within `dbshell` to find the proper one).
1. The user should be deleted correctly.
